src/common/components/registerAccountModal/registerAccountModal.component.js

```diff
@@ -196,6 +179,19 @@ class RegisterAccountModalController {
     modal.removeClass('modal-sm modal-md modal-lg')
     modal.addClass(`modal-${size}`)
   }
+
+  redirectToOktaForLogin () {
+    this.sessionService.signIn(this.lastPurchaseId).subscribe(() => {
+      const $injector = this.$injector
```

Comment:
Since the injector is only used once, can we create it here instead of using an instance variable? That minimizes the distance between where the variable is defined and where it is used and makes it easier to read the code.

```suggestion
      const $injector = angular.injector()
```

Reply:
Good point